charts/argo-rollouts/README.md

```diff
@@ -33,6 +33,16 @@ If dashboard is installed by `--set dashboard.enabled=true`, checkout the argo-r
 |:---------------------------------------------------------------------------|
 | The chart provides an option to change the service type (`dashboard.service.type`). Also it provides the ability to expose the dashboard via Ingress. Dashboard was never intended to be exposed as an administrative console -- it started out as a local view available via CLI. It should be protected by something (e.g. network access or even better an oauth proxy). |
 
+## Changelog
+
+For full list of changes please check ArtifactHub [changelog].
+
+Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
+
+### 2.22.3
+There was an unintentional change in behavior related to service creation with experiments introduced in v1.4.0 this has been reverted in v1.4.1 back to the original behavior. In v1.4.0 services where always created with in step based experiments even if there was no weight set. In 1.4.1 we go back to the original behavior of requiring weight to be set in order to create a service.
+For more information: [v1.4.1 release note](https://github.com/argoproj/argo-rollouts/releases/tag/v1.4.1).
+
```

Comment:
Upstream has a breaking change, so maybe it will help users to put a note. 🤔 

https://github.com/argoproj/argo-rollouts/releases/tag/v1.4.1

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/30188755/222450627-4ca411ff-2081-450d-8f1a-201001dc85e7.png">